### DISCUSSION TITLE
What is this character?

### DISCUSSION THREAD

**Discussion by ghost** [opendcs/opendcs#274] [Q&A] [answered]
Saw this weird character in DCP Message browser
![image](https://github.com/opendcs/opendcs/assets/140545535/ad807373-7f28-4bce-be77-a19235f2ea10)

Here is how it looks like commandline using `getDcpMessages.bat`
![image](https://github.com/opendcs/opendcs/assets/140545535/fda803cc-4511-42b7-a4af-5f76b3bac115)

And here is what it looks like in GOES field test website (https://dcs1.noaa.gov/Account/FieldTest)
![image](https://github.com/opendcs/opendcs/assets/140545535/866b12e2-1dbe-4f51-8ec1-e1ca17e0c24d)

Is this an equipment issue or Open DCS issue?

**Comment by MikeNeilson** [comment, marked as answer]
It's how the GUI is showing a "non-printable" character. That happens sometimes with certain GOES messages. I can't think of anything that would use one by default but it certainly happen if there's some sort of transmission error.

**Reply by MikeNeilson** [reply]
If you want the exact value you'll have to get the base64 encoded message out of the archive and look at it in a HEX editor.

**Reply by ghost** [reply]
So `$` is not the exact value?
